src/custom/state/ethFlow/atoms.ts

```diff
@@ -0,0 +1,8 @@
+import { selectAtom } from 'jotai/utils'
+import { atomWithStorage } from './atoms/utils'
+
+/**
+ * Base atom that stores the user's set native ETH sell value
+ */
+export const userNativeEthFlow = atomWithStorage<boolean>('userNativeEthFlow', true)
```

Comment:
creates an atom state prop held in storage (using custom atomWithStorage logic)

Reply:
Do you actually mean `user` rather than `use` or `uses` ?

Reply:
Also, when is this value set?
Unless I missed it, it looks like the value is only set on initialization.
If that's the case, why not a constant?

Reply:
Disregard the last comment. Waterfall PR, yada yada, I got it